### PR TITLE
Adding CA Policy Removed Sec Ops Rule

### PR DESCRIPTION
**Pull request by Corissalea**
Conditional Access Policy removed by non-approved actors

### CODE REVIEW

**Comment by frack113** [comment]
File was in windows `CR+LF` 
So Chance it to linux `LF` 
image

**Comment by Corissalea** [comment]
@frack113  - Got it, let me try that. Thank you!

> File was in windows `CR+LF` So Chance it to linux `LF` image

File: rules/cloud/azure/azure_aad_secops_ca_policy_updatedby_bad_actor.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+date: 2022/07/19
+references:
+  - https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/security-operations-infrastructure#conditional-access
+logsource:
+  product: azure
+  service: auditlogs
+detection:
+  keywords:
+    - Update conditional access policy
+  condition: selection
```
Comment (nasbench) [inline rules/cloud/azure/azure_aad_secops_ca_policy_updatedby_bad_actor.yml:14]: "keywords" must be used in the condition. Change the "condition" part to be 

`condition: keywords`

Comment (Neo23x0) [reply rules/cloud/azure/azure_aad_secops_ca_policy_updatedby_bad_actor.yml]: fixed it for her

**Pull request merged**